sql/moz-fx-data-shared-prod/telemetry_derived/newtab_desktop_daily_engagement_by_tile_id_position_v1/query.sql

```diff
@@ -0,0 +1,165 @@
+WITH
+  -- LEGACY Query
+legacy_deduplicated_pings AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.activity_stream_live.impression_stats_v1`
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+  QUALIFY
+    ROW_NUMBER() OVER (
+      PARTITION BY
+        DATE(submission_timestamp),
+        document_id
+      ORDER BY
+        submission_timestamp DESC
+    ) = 1
+),
+legacy_impression_data AS (
+  SELECT
+    d.*
+  FROM
+    legacy_deduplicated_pings AS d
+  WHERE
+    loaded IS NULL -- don't include loaded ping
+    -- include only data from Firefox > 121
+    AND SAFE_CAST(SPLIT(version, '.')[0] AS int64) <= 120
+    -- ensure data is valid/non-empty
+    AND ARRAY_LENGTH(tiles) >= 1
+    -- exclude custom Newtab page for Fx China
+    AND (page IS NULL OR page != 'https://newtab.firefoxchina.cn/newtab/as/activity-stream.html')
+),
+legacy_flattened_impression_data AS (
+  SELECT
+    submission_timestamp,
+    impression_id AS client_id,
+    flattened_tiles.id AS tile_id,
+    IFNULL(flattened_tiles.pos, alt_pos) AS position,
+    SUM(
+      CASE
+        WHEN click IS NULL
+          AND block IS NULL
+          AND pocket IS NULL
+          THEN 1
+        ELSE 0
+      END
+    ) AS impressions,
+    SUM(CASE WHEN click IS NOT NULL THEN 1 ELSE 0 END) AS clicks,
+    SUM(CASE WHEN pocket IS NOT NULL THEN 1 ELSE 0 END) AS pocketed,
+    SUM(CASE WHEN block IS NOT NULL THEN 1 ELSE 0 END) AS blocked
+  FROM
+    legacy_impression_data
+  CROSS JOIN
+    UNNEST(legacy_impression_data.tiles) AS flattened_tiles
+    WITH OFFSET AS alt_pos
+  GROUP BY
+    submission_timestamp,
+    client_id,
+    tile_id,
+    position
+),
+legacy_summary AS (
+  SELECT
+    submission_timestamp,
```

Comment:
This should be submission_date, no?

```suggestion
    DATE(submission_timestamp) AS submission_date,
```